recipe/meta.yaml

```diff
@@ -1,17 +1,16 @@
-{% set name = "flask" %}
-{% set version = "2.3.3" %}
+{% set version = "3.0.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: flask
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flask-{{ version }}.tar.gz
```

Comment:
I would slightly prefer to leave these as they were, since this is how `grayskull` produces the recipe and that makes it quite a bit easier to use `grayskull` to determine if the recipe has changed since the last release.

Reply:
Thank you for updating the recipe!

Reply:
updated. please feel free to push to this, or backport the changes to the other PR, as i likely won't have more time to put towards this today.

just trying to help out from a @conda-forge/werkzeug perspective, not a `{{ name }}` user myself.

Reply:
Thanks for changing this back.